Makefile

```diff
@@ -18,7 +18,7 @@
 export CGO_ENABLED=0
 
 # Details of the model to use:
-model_version:=v0.0.276
+model_version:=v0.0.277
```

Comment:
I think you could remove the version here.

@ciaranRoche we typically separate the changes and let to bump the version when we are going to release, right?

Reply:
No, this looks right, its only a model bump, you need to bump this var to run `make generate` to get the changes in.

Reply:
Thanks for taking a look @lucasponce , @lucasponce . 

When this one merged, I will create another MR with create the release.

Reply:
It looks like someone else already updated the model (the release v0.0.278 is already available).

I will close this PR.